WChat/src/app/screens/contactList/index.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import contacts from '../../../../resources/contacts.json';
+import Contact from './components/Contact';
+import {
+  View,
+  SectionList,
+  ListItem,
+  Text,
+  StyleSheet
+} from 'react-native';
+import styles from './styles'
+
+console.log(contacts);
+let asd = [{id: 'pe'}];
+export default class ContactList extends Component {
+  render() {
+    
+    const createContactsHeader = ({section}) => 
```

Comment:
define this function outside the render method. You can define it as an arrow function in the class, or outside the class!

Reply:
i prefer outside the class, above the `export default class ContactList extends Component {` line